src/util/http/handleError.ts

```diff
@@ -51,7 +51,7 @@ export function* handelError({
   yield put(
     httpResponseAction(actionName as RequestKeyExclude, {
       httpResponseStatus: 'error',
-      error: JSON.parse(JSON.stringify(e?.response || {})),
+      error: e.response,
```

Comment:
Here we select the `.response` property from the `e` object. So the `error` property will be of type `AxiosResponse` and not `AxiosError`.

Also, I removed the JSON.parse(JSON.stringify()) conversions which seemed to be absurd.